### ISSUE TITLE
BlueBerries

### ISSUE THREAD

**Issue by gohkenytp** [purejosh/sweetberryfoods#1]
What if Blueberries were added to this mod

**Comment by purejosh** [comment]
That's quite a decent idea! I'll play around with some ideas when I get home.

**Comment by purejosh** [comment]
I have successfully developed about 70% of Blueberries, which are almost identical to Sweet Berries in most ways. Blueberry bushes don't have thorns, so they do no damage, but they do slow you down. 

Things that are working, essentially 100% true to vanilla. 

✅ World generation
✅ Planting
✅ randomTick growing
✅ Loot drops when broken
✅ Loot tables for Taiga villages
✅ Slowing down entities

The worst is yet to come, though... 

❌ Bone meal growing
❌ Right click to harvest
❌ Composting
❌ Breeding/taming foxes

I'll update this ticket when I've made more progress. 

![image](https://user-images.githubusercontent.com/70633949/191667377-940a1875-08d7-409e-b372-80be0e7527d6.png)
